pkgs/development/libraries/malcontent/default.nix

```
@@ -101,6 +101,6 @@ stdenv.mkDerivation rec {
     homepage = "https://gitlab.freedesktop.org/pwithnall/malcontent";
     license = licenses.lgpl21Plus;
     maintainers = with maintainers; [ jtojnar ];
-    platforms = platforms.unix;
+    inherit (polkit.meta) platforms badPlatforms;
```

I am not sure how related this project to polkit is.

```suggestion
    platforms = platforms.linux;
    badPlatforms = [
      # mandatory libpolkit-gobject shared library
      lib.systems.inspect.platformPatterns.isStatic
    ];
```
Is the comment still correct?

---

> I am not sure how related this project to polkit is.

Heavily, as I explained in the commit message.

---

The comment wouldn't really be correct in this package because it's polkit that insists on building that library — would be confusing to mention it for this package.